app/controllers/admin/versions_controller.rb

```diff
@@ -0,0 +1,53 @@
+module Admin
+  class VersionsController < Admin::BaseController
+    skip_authorization_check
+
+    def index
+      authorize! :index, PaperTrail::Version.new(item_type: 'User')
+      conf_ids_for_organizer = current_user.is_admin? ? Conference.pluck(:id) : Conference.with_role(:organizer, current_user).pluck(:id)
+      @versions = PaperTrail::Version.where(["conference_id IN (?) OR item_type = 'User'", conf_ids_for_organizer]).order('created_at DESC')
+    end
+
+    def revert
+      @version = PaperTrail::Version.find(params[:id])
+      authorize! :revert, @version
+
+      if params[:attribute]
+        if @version.event != 'update'
+          flash[:error] = 'Reverting a specific attribute is allowed only for update type changes'
+
+        elsif @version.item.nil?
+          flash[:error] = 'Reverting a specific attribute is not allowed for deleted objects'
+
+        elsif @version.changeset.reject{ |_, values| values[0].blank? && values[1].blank? }.keys.include?(params[:attribute])
+          if @version.item[params[:attribute]] == @version.changeset[params[:attribute]][0]
+            flash[:error] = 'The item is already in the state that you are trying to revert it back to'
+          else
+            @version.item[params[:attribute]] = @version.changeset[params[:attribute]][0]
+            @version.item.save!
```

Comment:
I've added a check to see if save fails. But adding those conditions is raising this rubocop [offence](https://travis-ci.org/openSUSE/osem/builds/145311507#L1796).

I was thinking of splitting up this controller action.One for restoring the object to the version and once action for revert a single attribute.This would make it easier to define ability and will fix the rubocop offence mentioned above